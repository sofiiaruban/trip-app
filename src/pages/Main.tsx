import { FC } from 'react'
import { MAIN_PAGE_TEST_ID } from './constants'

const Main: FC = () => {
  return <div data-testid={MAIN_PAGE_TEST_ID}>Main</div>
}

export default Main
