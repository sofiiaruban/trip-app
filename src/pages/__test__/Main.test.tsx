import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Main from '../Main'

test('Renders the main page', () => {
  const { container } = render(<Main />)
  expect(container).toMatchSnapshot()
})
