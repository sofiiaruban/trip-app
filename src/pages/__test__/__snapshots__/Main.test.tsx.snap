// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the main page 1`] = `
<div>
  <div
    data-testid="mainPage"
  >
    Main
  </div>
</div>
`;
